src/components/form/form.stories.tsx

```diff
@@ -21,73 +20,19 @@ import isChromatic from "../../../.storybook/isChromatic";
 const isOpenForChromatic = isChromatic();
 
 export const DefaultWithStickyFooter = () => (
```

Comment:
Comment: There seems to be a lot of changes to the stories here that go beyond the scope of this ticket imo. I don't think we should be removing the tabs for example as they were added to catch regressions after issues were raised. We will move them away from the public docs at some point but I don't think it should be done now